pulp_ansible/app/tasks/collections.py

```diff
@@ -504,13 +465,14 @@ def __init__(self, remote, repository, is_repo_remote, deprecation_before_sync,
         self.already_synced = set()
         self._unpaginated_collection_metadata = None
         self._unpaginated_collection_version_metadata = None
+        self.optimize = optimize
```

Comment:
I think `self.is_repo_remote` is no longer used throughout the class looking at `_should_we_sync()`. If it is we can remove it.